torch/_dynamo/variables/tensor.py

```diff
@@ -1019,6 +1019,36 @@ def call_method(
     def python_type(self):
         return np.ndarray
 
+    def unpack_var_sequence(self, tx, idxes=None):
+        # XXX: a clone of TensorVariable method, w/ wrap_fx_proxy -> wrap_fx_proxy_cls
```

Comment:
not cool. Modify the TensorVariable method to call wrap_fx_proxy_cls with `type(self)`.

Reply:
Srlsly? lol, OK, fine. done.